src/main/resources/anniv5Resources/localization/eng/cosmoscommandpack/Cardstrings.json

```diff
@@ -21,7 +21,7 @@
   },
   "${ModID}:Recall": {
     "NAME": "Recall",
-    "DESCRIPTION": "Whenever a card is ${ModID}:Amplified, put a card from your Exhaust pile into your hand."
+    "DESCRIPTION": "The first time a card is ${ModID}:Amplified each turn, put a card from your Exhaust pile into your hand."
```

Comment:
Would it make more sense to have this stack by affecting the first X cards that Amplify each turn? That's how the base game effects for "the first/next time" stack. I'm mostly thinking of Echo Form and other duplication effects (you don't play the first card X times, you play the first card X cards twice), but this is also true for things like the double damage effect from Phantasmal Killer (you don't get triple damage, you get double damage for more turns).

You can probably find a counter-example somewhere, but in most cases where increasing the number of cards affected or extending the duration makes sense, that's what stacking a power does.

Reply:
That sounds sensible. Question though: do you think the power should "activate" and count up even if the Exhaust pile is empty?

Reply:
Good question. It seems to me that the case where your exhaust pile is empty is uncommon, and this would only affect the case where you have this power applied, you've exhausted nothing yet (even though you are probably drafting cards to take advantage of this), and you have both an exhaust card and amplify card in hand.

To me, that says that the best thing to do is to keep this simple -- extra complexity for a relatively niche case doesn't seem like its worth it.

If you did want to skip the activation when the exhaust pile was empty, IMO it should be stated on the card/power -- e.g., "The first time a card is Amplified each turn while your exhaust pile isn't empty, ..."